startup/grpcclient.go

```diff
@@ -0,0 +1,7 @@
+package startup
+
+import (
+	"github.com/rkvst/go-rkvstcommon/grpcclient"
+)
+
```

Comment:
Just a question: Why is this type alias pattern considered good ?

Reply:
its a common idiom in most packages and means that we only have to import grpcclient once - it also documents the interface (that could be mocked for tests if wanted)

Reply:
it was originally introduced to make it easier to port code to another API but is also commonly used to 'promote' an internal definition so that is is externally visible.

For example: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azservicebus/error.go